pkg/sdk/procedures_dto_builders_gen.go

```diff
@@ -0,0 +1,694 @@
+// Code generated by dto builder generator; DO NOT EDIT.
```

Comment:
There should be some required attributes added to def, so this file will also change after regeneration